modules/nw-ingress-controller-configuration-parameters.adoc

```diff
@@ -138,6 +138,7 @@ The `AllowedSubjectPatterns` is an optional value that specifies a list of regul
 *** `syslog` describes parameters for the `Syslog` logging destination type:
 **** `address` is the IP address of the syslog endpoint that receives log messages.
 **** `port` is the UDP port number of the syslog endpoint that receives log messages.
+**** `maxLength` is the maximum length of the syslog message. The maximum length is 1024 by default. If this field is empty, the maximum length is set to the default value.
```

Comment:
There is  a range for syslog message length. 480-4096, you may include that also

Reply:
@melvinjoseph86 Comment addressed.